table/full_filter_block.cc

```diff
@@ -147,4 +151,51 @@ bool FullFilterBlockReader::MayMatch(const Slice& entry) {
 size_t FullFilterBlockReader::ApproximateMemoryUsage() const {
   return contents_.size();
 }
+
+bool FullFilterBlockReader::IsFilterCompatible(
+    const ReadOptions& read_options, const Slice& user_key,
+    const SliceTransform* table_prefix_extractor,
+    const Comparator* comparator) {
```

Comment:
Why not call it MayRangeExists() which returns results directly, rather than calling IsFilterCompatible() and then PrefixMayExists()?